specification.rst

```diff
@@ -2854,3 +2854,21 @@ The page abbreviation rules for the different values of the
 "minimal-two"
     As "minimal", but at least two digits are kept in the second number when it
     has two or more digits long.
+
+
+Appendix VI: Links
+------------------
+
+The CSL syntax does not have support for configuration of links. However,
+processors should include links on bibliographic references, using the following
+rules:
+
+If the item includes the following identifiers, the target of the link should be
+set, in order of priority:
+
+1. ``url``: output as is
+2. ``doi``: prepend with ``https://dx.doi.org/``
```

Comment:
Should be `prepend with ``https://doi.org/```

Reply:
I'm unsure of the priority that should be given to `doi`/`pmid`/`pmcid` vs. `url`. The first three are intended to be stable, unchanging identifiers that are more reliable than URLs, but some publishers (namely Taylor and Francis) are pretty bad at registering their DOIs and ensuring that they resolve in a timely manner.

Reply:
I was unsure also. 

I also considered whether to create separate lists.

Reply:
But I was hoping we could come up with some solid suggestions worth putting in the spec, but also leave room for iteration over time.

Reply:
@adam3smith What do you think regarding DOI vs URL prioirity?

Reply:
I'd say stable identifiers first.